cloudconfig/powershell_helpers.go

```diff
@@ -19,34 +28,34 @@ $ErrorActionPreference = "Stop"
 
 function ExecRetry($command, $maxRetryCount = 10, $retryInterval=2)
```

Comment:
I'm afraid I really don't like doing this. If we need code in another file I'd much rather have that code in a seperate file and read it when needed. If we need a package var I'd read it in a package init()

http://paste.ubuntu.com/12530761/